resources/views/backend/city/index.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('backend.layouts.master')
+
+@section('content')
+
+<!-- page content --> 
+<div class="right_col" role="main">
+    <div class=""> 
+        <div class="page-title"> 
+            <div class="title_left"> 
+                <h3>{!! trans('labels.city') !!} <small>{!! trans('labels.manager') !!}</small></h3> 
+            </div> 
+        </div> 
+
+        <div class="clearfix"></div> 
+        <div class="row"> 
+            <div class="col-md-12 col-sm-12 col-xs-12"> 
+                <div class="x_panel">
+                    @if (count($cities) > 0)
+                    <div class="x_title"> 
+                        <a class="btn btn-md btn-primary" href="{{ url('admin/city/create') }}">{!! trans('labels.add_new') !!}</a>
+                        <div class="clearfix"></div> 
+
+                        @if (Session::has('message'))
```

Comment:
please move this code to a partials view
Or you can use this [Package](https://github.com/laracasts/flash)